scripts/ci.hs

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env nix-shell
+#!nix-shell ../shell.nix -A ci --run "runhaskell -iiohk scripts/ci.hs"
```

Comment:
once testing is done, it should probably be switched to a nix derivation that is pre-built by hydra